doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,174 @@
 [<< Back](../)
 
 # 1. Introduction
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
+
+* [1.1 Introduction](#1.1)
+  * [1.1.1 Terminology](#1.1)
+  * [1.1.2 Relation to other communities](#1.1)
+* [1.2 Scope](#1.2)
+* [1.3 Principles and Guidelines](#1.3)
+  * [1.4.1 Overarching Objectives and Goals](#1.4.1)
+* [1.4 Best Practices](#1.4)
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+* [1.9 Resources & References](#1.9)
+
+## Executive Summary
+
+The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
```

Comment:
```suggestion
The Reference Conformance 2 (RC2) was established to ensure implementations of the Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
```

Reply:
expanding RC2 and removing CNTT is the suggestion

Reply:
@tomkivlin "... driven quality assurance standards for conformance, verification, and validation". One could say that verification and validation are used in evaluating conformance. Maybe, the sentence can end at "conformance"

For conformance not everything is testable -- some may require other methods of verification. The next sentence, can be changed, as in:
"For the purpose of this chapter, cloud infrastructure and cloud native network function (CNF) verification and validation will be performed to evaluate **Conformance** (i.e. adherence) of implementations, such as RI2, with RA2."

Reply:
Yes makes sense @pgoyal01